OpenRA.Mods.Common/Widgets/Logic/Editor/NewMapLogic.cs

```diff
@@ -68,26 +68,30 @@ public NewMapLogic(Action onExit, Action<string> onSelect, Ruleset modRules, Wid
 				map.PlayerDefinitions = new MapPlayers(map.Rules, map.SpawnPoints.Value.Length).ToMiniYaml();
 				map.FixOpenAreas(modRules);
 
-				var userMapFolder = Game.ModData.Manifest.MapFolders.First(f => f.Value == "User").Key;
+				Action<string> afterSave = uid =>
+				{
+					// HACK: Work around a synced-code change check.
```

Comment:
Not sure why you can't reproduce, but I am happy now as I just crashed with your older branch and now didn't with the latest commit added. :white_check_mark: